src/style.css

```diff
@@ -0,0 +1,151 @@
+html {
+  font-family: Roboto, Helvetika, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  width: 200px;
+  height: 406px;
+  box-sizing: border-box;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.container__logo {
+  width: 160px;
+  height: 134px;
+  padding: 32px 0 0 19px;
+}
+
+/* #region information */
+.conainer_information {
+  margin: 40px 16px 0;
+}
+
+.conainer_tovar {
+  font-weight: 500;
+  font-size: 12px;
+  margin-bottom: 4px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.conainer_kod-tovara {
+  font-weight: 400;
+  font-size: 10px;
+
+  line-height: 14px;
+  color: #616070;
+}
+/* #endregion */
+
+/* #region review */
+.container__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin: 16px 16px 0;
+}
+
+.container__stars {
+  height: 16px;
+  width: 96px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.container__star {
+  margin-right: 4px;
+  background-color: transparent;
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.container__star:last-child {
+  background-image: url(images/star.svg);
+  margin-right: 0;
+}
+
+.container__review-col {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+  line-height: 14px;
+}
+/* #endregion */
+
+/* #region buy */
+.container__buy {
+  display: flex;
+  justify-content: space-between;
+  margin: 24px 16px 0;
+}
+
+.price {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+  line-height: 18px;
+}
+
+.price-grn {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+  line-height: 18px;
+}
+/* #endregion */
+
+/* #region buy-link */
+.container__buy-link {
+  display: flex;
+  justify-content: center;
+  margin-top: 16px;
+}
+
+.buy-submit {
+  display: flex;
+  justify-content: center;
+  box-sizing: border-box;
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+  text-decoration: none;
+  height: 40px;
+  width: 166px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  padding-top: 11px;
+  line-height: 16px;
+}
+
+.buy-submit:hover {
+  display: flex;
+  justify-content: center;
+  box-sizing: border-box;
+  font-size: 14px;
+  font-weight: 700;
+  color: #00acdc;
+  text-decoration: none;
+  height: 40px;
+  width: 166px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
```

Comment:
Don't add new border to the element on hover. Add default transparent border of the same width.
```suggestion
```